okhttp/src/test/java/com/backblaze/b2/client/okHttpClient/B2StorageHttpClientFactoryTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019, Backblaze Inc. All Rights Reserved.
+ * License https://www.backblaze.com/using_b2_code.html
+ */
+package com.backblaze.b2.client.okHttpClient;
+
+import com.backblaze.b2.client.B2StorageClientFactory;
+import com.backblaze.b2.client.B2StorageClientFactoryPathBasedImpl;
+import org.junit.Test;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class B2StorageHttpClientFactoryTest {
```

Comment:
should this be  B2StorageOkHttpClientFactoryTest  (not the "ok" in the middle of the name)?